.gitignore

```diff
@@ -4,3 +4,7 @@ bin
 
 ## Intermediate build files
 build
+
+
+## Swap Files
+*.swp
```

Comment:
Swap files are temp files which vim makes when files are opened. I typically use multiple terminals with some files already open so these files show up in the git status. I'm not sure what the conventions are with .gitignore so if it doesn't fit the bill I'll remove it.